modules/library/api/src/main/java/org/geotools/geometry/jts/ReferencedEnvelope.java

```diff
@@ -875,21 +897,7 @@ public static ReferencedEnvelope reference(Envelope e) {
             return new ReferencedEnvelope(e, null);
         }
     }
-
```

Comment:
I thought having a specific BoundingBox method helped one of the compilers not freak out, requiring client code to add casting.